src/renate/benchmark/experiment_config.py

```diff
@@ -361,5 +359,6 @@ def lr_scheduler_fn(
     raise ValueError(f"Unknown scheduler `{learning_rate_scheduler}`.")
 
 
-def metrics_fn() -> Dict:
-    return {"accuracy": Accuracy()}
+def metrics_fn(num_outputs: int) -> Dict:
+    task = "binary" if num_outputs == 2 else "multiclass"
```

Comment:
I don't think this will work. IIUC, the binary accuracy in `torchmetrics` expects scalar predictions.

Reply:
good point. do we want to use `num_outputs==1` and account for this change in `loss_fn` and `model_fn`?